app/src/main/java/org/simple/clinic/drugs/selectionv2/dosage/DosagePickerSheet.kt

```diff
@@ -58,8 +59,8 @@ class DosagePickerSheet : BottomSheetActivity() {
   private fun sheetCreates(): Observable<UiEvent> {
     val drugName = intent.getStringExtra(KEY_DRUG_NAME)
     val patientUuid = intent.getSerializableExtra(KEY_PATIENT_UUID) as UUID
-    val prescribedDrugUuid = intent.getSerializableExtra(KEY_PRESCRIBED_DRUG_UUID) as UUID
-    return Observable.just(DosagePickerSheetCreated(drugName, patientUuid, prescribedDrugUuid.toOptional()))
+    var prescribedDrugUuid = intent.getSerializableExtra(KEY_PRESCRIBED_DRUG_UUID).toOptional() as Optional<UUID>
```

Comment:
Why is this a variable instead of a constant?

Reply:
Mistake.